core/src/test/java/io/questdb/griffin/TableBackupTest.java

```diff
@@ -124,7 +124,7 @@ public TimestampFormat getBackupDirTimestampFormat() {
         MessageBus mainMessageBus = new MessageBusImpl();
         mainEngine = new CairoEngine(mainConfiguration, mainMessageBus);
         mainCompiler = new SqlCompiler(mainEngine);
-        mainSqlExecutionContext = new SqlExecutionContextImpl(mainMessageBus, 1, mainEngine).with(AllowAllCairoSecurityContext.INSTANCE, new BindVariableService(), null);
+        mainSqlExecutionContext = new SqlExecutionContextImpl(mainMessageBus, 1, mainEngine).with(AllowAllCairoSecurityContext.INSTANCE, new BindVariableService(), null, -1, null);
```

Comment:
Case in point. Changes to SqlExecutionContext should have very limited impact on tests that are not using new features.